tests/test_smoke_elicit.py

```diff
@@ -0,0 +1,66 @@
+from pathlib import Path
+
+import pytest
+
+from elk import ExtractionConfig
+from elk.extraction import PromptConfig
+from elk.training import CcsReporterConfig, EigenReporterConfig
+from elk.training.train import train, RunConfig
+
+"""
+TODO: These tests should work with deberta
+but you'll need to make deberta fp32 instead of fp16
+because pytorch cpu doesn't support fp16
+"""
+
```

Comment:
basically the deberta model itself does some fp16 operations that aren't supported. but tiny-gpt doesn't. so that why tiny-gpt works. 

i'll try to figure that out in another MR. i think we could convert fp16 models to fp32 if its CPU, but seems hacky?

Reply:
I actually don't view this as hacky. In general float16 isn't supported on CPUs. We could check the whether we're running on CPU right when we call `AutoModel.from_pretrained` and tell HF to always load the model as float32 in that case, and use the dtype of the checkpoint otherwise.

Reply:
^ just made this change, so if you want to add a deberta test you can. I think tiny gpt2 is probably sufficient for now though